### ISSUE TITLE
PAXMiner is processsing some data for channels not tagged as AOs.

### ISSUE THREAD

**Issue by drewlarson1** [F3Nation-Community/slackblast#40]
**Background**
We maintain two 3rd-F locations as channels for 1) preblasts and 2) qsignups. We don't want these two channels processed as "workouts" by PAXminer. These two channels are NOT set as AOs in the database AND paxminer isn't in the channel.

**Issue**
PAXMiner appears to be processing the data from the backblast. These backblasts were entered using the /backblast command and have the buttons at the bottom. When I say "appears," I mean that PAXMiner is counting the Q stats for these channels and including it in the report on the 1st-F channel as well as the individual 3rd-F channels. I assume the same is true of the PAX listed therein.

**Resolution**
If the issue is caused by using the /backblast command, then ensure /backblast command is checking for the AO flag in the database before processing the backblast. Then again, this issue might not exist at all (can't remember if I recently fixed these flags in the database #old).

Griswold, F3xT

**Comment by farrellw** [comment]
@evanpetzoldt this ticket was over in PaxMiner. I saw the reference to the /backblast command, do you think it belongs here instead?

**Comment by evanpetzoldt** [comment]
Yes, thanks @farrellw , this is relevant here. While I'm not strictly opposed to this (and it wouldn't be terribly difficult to implement), I sort of set it up the way I did on purpose... By observation, I'm not fond of the approach of having to go into SQL and manually turn channels on and off... Often they don't realize it's not on for a while, the guy who has the credentials is on vacation, etc. It's just sort of clunky.

Of course, having a disconnect between data and pm reporting isn't great, so I could go either way.
